build.gradle

```diff
@@ -46,36 +46,111 @@ configurations {
                 url 'https://foursquare.jfrog.io/foursquare/libs-release/'
             }
         }
+        
+        exclude group: 'com.android.support', module: 'animated-vector-drawable'
```

Comment:
do you know where these are coming from? seems pretty concerning that wed have to do all of this work. You probably want to enable the jetifier to fix this for you.

in `gradle.properties` you can try adding

```
# Allows us to more easily migrate to AndroidX
# all of our dependencies
android.useAndroidX=true
android.enableJetifier=true
```

Reply:
i'll try that

Reply:
@kylefowler Looks like chucker needs the support libs, I'm able to remove most of them because it looks like we only use `com.android.support:cardview-v7:28.0.0` and `com.android.support:design:28.0.0` indirectly from chucker

```
pilgrim
+--- androidx.activity:activity:1.2.0-alpha06
|    +--- androidx.annotation:annotation:1.1.0
|    +--- androidx.core:core:1.1.0
|    |    +--- androidx.annotation:annotation:1.1.0
|    |    +--- androidx.lifecycle:lifecycle-runtime:2.0.0 -> 2.3.0-alpha04
|    |    |    +--- androidx.lifecycle:lifecycle-common:2.3.0-alpha04
|    |    |    |    \--- androidx.annotation:annotation:1.1.0
|    |    |    +--- androidx.arch.core:core-common:2.1.0
|    |    |    |    \--- androidx.annotation:annotation:1.1.0
|    |    |    \--- androidx.annotation:annotation:1.1.0
|    |    +--- androidx.versionedparcelable:versionedparcelable:1.1.0
|    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    \--- androidx.collection:collection:1.0.0 -> 1.1.0
|    |    |         \--- androidx.annotation:annotation:1.1.0
|    |    \--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    +--- androidx.lifecycle:lifecycle-runtime:2.3.0-alpha04 (*)
|    +--- androidx.lifecycle:lifecycle-viewmodel:2.3.0-alpha04
|    |    \--- androidx.annotation:annotation:1.1.0
|    +--- androidx.savedstate:savedstate:1.1.0-alpha01
|    |    +--- androidx.annotation:annotation:1.1.0
|    |    +--- androidx.arch.core:core-common:2.0.1 -> 2.1.0 (*)
|    |    \--- androidx.lifecycle:lifecycle-common:2.0.0 -> 2.3.0-alpha04 (*)
|    +--- androidx.lifecycle:lifecycle-viewmodel-savedstate:2.3.0-alpha04
|    |    +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    +--- androidx.savedstate:savedstate:1.0.0 -> 1.1.0-alpha01 (*)
|    |    +--- androidx.lifecycle:lifecycle-livedata-core:2.3.0-alpha04
|    |    |    +--- androidx.arch.core:core-common:2.1.0 (*)
|    |    |    +--- androidx.arch.core:core-runtime:2.1.0
|    |    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    |    \--- androidx.arch.core:core-common:[2.1.0] -> 2.1.0 (*)
|    |    |    \--- androidx.lifecycle:lifecycle-common:2.3.0-alpha04 (*)
|    |    \--- androidx.lifecycle:lifecycle-viewmodel:2.3.0-alpha04 (*)
|    \--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
+--- com.foursquare:pilgrimsdk:2.4.2
|    +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.61
|    |    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.3.61
|    |    \--- org.jetbrains:annotations:13.0
|    +--- org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.3.61
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.3.61 (*)
|    +--- com.google.code.gson:gson:2.8.2 -> 2.8.5
|    +--- com.squareup.okhttp3:okhttp:3.9.0 -> 3.12.2
|    |    \--- com.squareup.okio:okio:1.15.0
|    +--- com.google.android.gms:play-services-ads-identifier:17.0.0
|    |    \--- com.google.android.gms:play-services-basement:17.0.0
|    |         +--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    |         +--- androidx.core:core:1.0.0 -> 1.1.0 (*)
|    |         \--- androidx.fragment:fragment:1.0.0 -> 1.1.0
|    |              +--- androidx.annotation:annotation:1.1.0
|    |              +--- androidx.core:core:1.1.0 (*)
|    |              +--- androidx.collection:collection:1.1.0 (*)
|    |              +--- androidx.viewpager:viewpager:1.0.0
|    |              |    +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |              |    +--- androidx.core:core:1.0.0 -> 1.1.0 (*)
|    |              |    \--- androidx.customview:customview:1.0.0
|    |              |         +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |              |         \--- androidx.core:core:1.0.0 -> 1.1.0 (*)
|    |              +--- androidx.loader:loader:1.0.0
|    |              |    +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |              |    +--- androidx.core:core:1.0.0 -> 1.1.0 (*)
|    |              |    +--- androidx.lifecycle:lifecycle-livedata:2.0.0 -> 2.1.0
|    |              |    |    +--- androidx.arch.core:core-runtime:2.1.0 (*)
|    |              |    |    +--- androidx.lifecycle:lifecycle-livedata-core:2.1.0 -> 2.3.0-alpha04 (*)
|    |              |    |    \--- androidx.arch.core:core-common:2.1.0 (*)
|    |              |    \--- androidx.lifecycle:lifecycle-viewmodel:2.0.0 -> 2.3.0-alpha04 (*)
|    |              +--- androidx.activity:activity:1.0.0 -> 1.2.0-alpha06 (*)
|    |              \--- androidx.lifecycle:lifecycle-viewmodel:2.0.0 -> 2.3.0-alpha04 (*)
|    +--- com.google.android.gms:play-services-base:17.0.0
|    |    +--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    |    +--- androidx.core:core:1.0.0 -> 1.1.0 (*)
|    |    +--- androidx.fragment:fragment:1.0.0 -> 1.1.0 (*)
|    |    +--- com.google.android.gms:play-services-basement:17.0.0 (*)
|    |    \--- com.google.android.gms:play-services-tasks:17.0.0
|    |         \--- com.google.android.gms:play-services-basement:17.0.0 (*)
|    +--- com.google.android.gms:play-services-basement:17.0.0 (*)
|    +--- com.google.android.gms:play-services-location:17.0.0
|    |    +--- com.google.android.gms:play-services-base:17.0.0 (*)
|    |    +--- com.google.android.gms:play-services-basement:17.0.0 (*)
|    |    +--- com.google.android.gms:play-services-places-placereport:17.0.0
|    |    |    \--- com.google.android.gms:play-services-basement:17.0.0 (*)
|    |    \--- com.google.android.gms:play-services-tasks:17.0.0 (*)
|    +--- com.google.android.gms:play-services-gcm:17.0.0
|    |    +--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    |    +--- androidx.core:core:1.0.0 -> 1.1.0 (*)
|    |    +--- androidx.legacy:legacy-support-core-utils:1.0.0
|    |    |    +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    +--- androidx.core:core:1.0.0 -> 1.1.0 (*)
|    |    |    +--- androidx.documentfile:documentfile:1.0.0
|    |    |    |    \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    +--- androidx.loader:loader:1.0.0 (*)
|    |    |    +--- androidx.localbroadcastmanager:localbroadcastmanager:1.0.0
|    |    |    |    \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    \--- androidx.print:print:1.0.0
|    |    |         \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    +--- com.google.android.gms:play-services-base:17.0.0 (*)
|    |    +--- com.google.android.gms:play-services-basement:17.0.0 (*)
|    |    +--- com.google.android.gms:play-services-iid:[17.0.0] -> 17.0.0
|    |    |    +--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    |    |    +--- androidx.core:core:1.0.0 -> 1.1.0 (*)
|    |    |    +--- com.google.android.gms:play-services-base:17.0.0 (*)
|    |    |    +--- com.google.android.gms:play-services-basement:17.0.0 (*)
|    |    |    +--- com.google.android.gms:play-services-stats:17.0.0
|    |    |    |    +--- androidx.legacy:legacy-support-core-utils:1.0.0 (*)
|    |    |    |    \--- com.google.android.gms:play-services-basement:17.0.0 (*)
|    |    |    \--- com.google.android.gms:play-services-tasks:17.0.0 (*)
|    |    \--- com.google.android.gms:play-services-stats:17.0.0 (*)
|    +--- com.evernote:android-job:1.4.2
|    |    \--- androidx.core:core:1.1.0 (*)
|    \--- androidx.work:work-runtime-ktx:2.3.0
|         +--- androidx.work:work-runtime:[2.3.0] -> 2.3.0
|         |    +--- com.google.guava:listenablefuture:1.0
|         |    +--- androidx.lifecycle:lifecycle-livedata:2.1.0 (*)
|         |    +--- androidx.room:room-runtime:2.2.2
|         |    |    +--- androidx.room:room-common:[2.2.2] -> 2.2.2
|         |    |    |    \--- androidx.annotation:annotation:1.1.0
|         |    |    +--- androidx.sqlite:sqlite-framework:2.0.1
|         |    |    |    +--- androidx.annotation:annotation:1.0.2 -> 1.1.0
|         |    |    |    \--- androidx.sqlite:sqlite:2.0.1
|         |    |    |         \--- androidx.annotation:annotation:1.0.2 -> 1.1.0
|         |    |    +--- androidx.sqlite:sqlite:2.0.1 (*)
|         |    |    \--- androidx.arch.core:core-runtime:2.0.1 -> 2.1.0 (*)
|         |    +--- androidx.core:core:1.1.0 (*)
|         |    \--- androidx.lifecycle:lifecycle-service:2.1.0
|         |         \--- androidx.lifecycle:lifecycle-runtime:2.1.0 -> 2.3.0-alpha04 (*)
|         +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.60 -> 1.3.61 (*)
|         \--- org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.0
|              +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.50 -> 1.3.61 (*)
|              \--- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.0
|                   +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.50 -> 1.3.61 (*)
|                   \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.3.50 -> 1.3.61
\--- com.foursquare:pilgrimsdk-debugging:2.4.2
     +--- androidx.appcompat:appcompat:1.1.0
     |    +--- androidx.annotation:annotation:1.1.0
     |    +--- androidx.core:core:1.1.0 (*)
     |    +--- androidx.cursoradapter:cursoradapter:1.0.0
     |    |    \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
     |    +--- androidx.fragment:fragment:1.1.0 (*)
     |    +--- androidx.appcompat:appcompat-resources:1.1.0
     |    |    +--- androidx.annotation:annotation:1.1.0
     |    |    +--- androidx.core:core:1.0.1 -> 1.1.0 (*)
     |    |    +--- androidx.vectordrawable:vectordrawable:1.1.0
     |    |    |    +--- androidx.annotation:annotation:1.1.0
     |    |    |    +--- androidx.core:core:1.1.0 (*)
     |    |    |    \--- androidx.collection:collection:1.1.0 (*)
     |    |    +--- androidx.vectordrawable:vectordrawable-animated:1.1.0
     |    |    |    +--- androidx.vectordrawable:vectordrawable:1.1.0 (*)
     |    |    |    +--- androidx.interpolator:interpolator:1.0.0
     |    |    |    |    \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
     |    |    |    \--- androidx.collection:collection:1.1.0 (*)
     |    |    \--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
     |    +--- androidx.drawerlayout:drawerlayout:1.0.0
     |    |    +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
     |    |    +--- androidx.core:core:1.0.0 -> 1.1.0 (*)
     |    |    \--- androidx.customview:customview:1.0.0 (*)
     |    \--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
     +--- androidx.recyclerview:recyclerview:1.0.0
     |    +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
     |    +--- androidx.core:core:1.0.0 -> 1.1.0 (*)
     |    \--- androidx.legacy:legacy-support-core-ui:1.0.0
     |         +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
     |         +--- androidx.core:core:1.0.0 -> 1.1.0 (*)
     |         +--- androidx.legacy:legacy-support-core-utils:1.0.0 (*)
     |         +--- androidx.customview:customview:1.0.0 (*)
     |         +--- androidx.viewpager:viewpager:1.0.0 (*)
     |         +--- androidx.coordinatorlayout:coordinatorlayout:1.0.0
     |         |    +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
     |         |    +--- androidx.core:core:1.0.0 -> 1.1.0 (*)
     |         |    \--- androidx.customview:customview:1.0.0 (*)
     |         +--- androidx.drawerlayout:drawerlayout:1.0.0 (*)
     |         +--- androidx.slidingpanelayout:slidingpanelayout:1.0.0
     |         |    +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
     |         |    +--- androidx.core:core:1.0.0 -> 1.1.0 (*)
     |         |    \--- androidx.customview:customview:1.0.0 (*)
     |         +--- androidx.interpolator:interpolator:1.0.0 (*)
     |         +--- androidx.swiperefreshlayout:swiperefreshlayout:1.0.0
     |         |    +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
     |         |    +--- androidx.core:core:1.0.0 -> 1.1.0 (*)
     |         |    \--- androidx.interpolator:interpolator:1.0.0 (*)
     |         +--- androidx.asynclayoutinflater:asynclayoutinflater:1.0.0
     |         |    +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
     |         |    \--- androidx.core:core:1.0.0 -> 1.1.0 (*)
     |         \--- androidx.cursoradapter:cursoradapter:1.0.0 (*)
     +--- androidx.constraintlayout:constraintlayout:1.1.3
     |    \--- androidx.constraintlayout:constraintlayout-solver:1.1.3
     +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.61 (*)
     +--- com.github.chuckerteam.chucker:library:3.0.0
     |    +--- org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.3.41 -> 1.3.61 (*)
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.41
     |    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.3.41 -> 1.3.61 (*)
     |    +--- com.google.code.gson:gson:2.8.5
     |    +--- com.squareup.okhttp3:okhttp:3.12.2 (*)
     |    +--- com.android.support:design:28.0.0
     |    |    +--- com.android.support:support-annotations:28.0.0
     |    |    +--- com.android.support:support-compat:28.0.0
     |    |    |    +--- com.android.support:support-annotations:28.0.0
     |    |    |    +--- com.android.support:collections:28.0.0
     |    |    |    |    \--- com.android.support:support-annotations:28.0.0
     |    |    |    +--- android.arch.lifecycle:runtime:1.1.1
     |    |    |    |    +--- android.arch.lifecycle:common:1.1.1
     |    |    |    |    |    \--- com.android.support:support-annotations:26.1.0 -> 28.0.0
     |    |    |    |    +--- android.arch.core:common:1.1.1
     |    |    |    |    |    \--- com.android.support:support-annotations:26.1.0 -> 28.0.0
     |    |    |    |    \--- com.android.support:support-annotations:26.1.0 -> 28.0.0
     |    |    |    \--- com.android.support:versionedparcelable:28.0.0
     |    |    |         +--- com.android.support:support-annotations:28.0.0
     |    |    |         \--- com.android.support:collections:28.0.0 (*)
     |    |    +--- com.android.support:support-core-ui:28.0.0
     |    |    |    +--- com.android.support:support-annotations:28.0.0
     |    |    |    +--- com.android.support:support-compat:28.0.0 (*)
     |    |    |    +--- com.android.support:support-core-utils:28.0.0
     |    |    |    |    +--- com.android.support:support-annotations:28.0.0
     |    |    |    |    +--- com.android.support:support-compat:28.0.0 (*)
     |    |    |    |    +--- com.android.support:documentfile:28.0.0
     |    |    |    |    |    \--- com.android.support:support-annotations:28.0.0
     |    |    |    |    +--- com.android.support:loader:28.0.0
     |    |    |    |    |    +--- com.android.support:support-annotations:28.0.0
     |    |    |    |    |    +--- com.android.support:support-compat:28.0.0 (*)
     |    |    |    |    |    +--- android.arch.lifecycle:livedata:1.1.1
     |    |    |    |    |    |    +--- android.arch.core:runtime:1.1.1
     |    |    |    |    |    |    |    +--- com.android.support:support-annotations:26.1.0 -> 28.0.0
     |    |    |    |    |    |    |    \--- android.arch.core:common:1.1.1 (*)
     |    |    |    |    |    |    +--- android.arch.lifecycle:livedata-core:1.1.1
     |    |    |    |    |    |    |    +--- android.arch.lifecycle:common:1.1.1 (*)
     |    |    |    |    |    |    |    +--- android.arch.core:common:1.1.1 (*)
     |    |    |    |    |    |    |    \--- android.arch.core:runtime:1.1.1 (*)
     |    |    |    |    |    |    \--- android.arch.core:common:1.1.1 (*)
     |    |    |    |    |    \--- android.arch.lifecycle:viewmodel:1.1.1
     |    |    |    |    |         \--- com.android.support:support-annotations:26.1.0 -> 28.0.0
     |    |    |    |    +--- com.android.support:localbroadcastmanager:28.0.0
     |    |    |    |    |    \--- com.android.support:support-annotations:28.0.0
     |    |    |    |    \--- com.android.support:print:28.0.0
     |    |    |    |         \--- com.android.support:support-annotations:28.0.0
     |    |    |    +--- com.android.support:customview:28.0.0
     |    |    |    |    +--- com.android.support:support-annotations:28.0.0
     |    |    |    |    \--- com.android.support:support-compat:28.0.0 (*)
     |    |    |    +--- com.android.support:viewpager:28.0.0
     |    |    |    |    +--- com.android.support:support-annotations:28.0.0
     |    |    |    |    +--- com.android.support:support-compat:28.0.0 (*)
     |    |    |    |    \--- com.android.support:customview:28.0.0 (*)
     |    |    |    +--- com.android.support:coordinatorlayout:28.0.0
     |    |    |    |    +--- com.android.support:support-annotations:28.0.0
     |    |    |    |    +--- com.android.support:support-compat:28.0.0 (*)
     |    |    |    |    \--- com.android.support:customview:28.0.0 (*)
     |    |    |    +--- com.android.support:drawerlayout:28.0.0
     |    |    |    |    +--- com.android.support:support-annotations:28.0.0
     |    |    |    |    +--- com.android.support:support-compat:28.0.0 (*)
     |    |    |    |    \--- com.android.support:customview:28.0.0 (*)
     |    |    |    +--- com.android.support:slidingpanelayout:28.0.0
     |    |    |    |    +--- com.android.support:support-annotations:28.0.0
     |    |    |    |    +--- com.android.support:support-compat:28.0.0 (*)
     |    |    |    |    \--- com.android.support:customview:28.0.0 (*)
     |    |    |    +--- com.android.support:interpolator:28.0.0
     |    |    |    |    \--- com.android.support:support-annotations:28.0.0
     |    |    |    +--- com.android.support:swiperefreshlayout:28.0.0
     |    |    |    |    +--- com.android.support:support-annotations:28.0.0
     |    |    |    |    +--- com.android.support:support-compat:28.0.0 (*)
     |    |    |    |    \--- com.android.support:interpolator:28.0.0 (*)
     |    |    |    +--- com.android.support:asynclayoutinflater:28.0.0
     |    |    |    |    +--- com.android.support:support-annotations:28.0.0
     |    |    |    |    \--- com.android.support:support-compat:28.0.0 (*)
     |    |    |    \--- com.android.support:cursoradapter:28.0.0
     |    |    |         \--- com.android.support:support-annotations:28.0.0
     |    |    +--- com.android.support:support-core-utils:28.0.0 (*)
     |    |    +--- com.android.support:support-fragment:28.0.0
     |    |    |    +--- com.android.support:support-compat:28.0.0 (*)
     |    |    |    +--- com.android.support:support-core-ui:28.0.0 (*)
     |    |    |    +--- com.android.support:support-core-utils:28.0.0 (*)
     |    |    |    +--- com.android.support:support-annotations:28.0.0
     |    |    |    +--- com.android.support:loader:28.0.0 (*)
     |    |    |    \--- android.arch.lifecycle:viewmodel:1.1.1 (*)
     |    |    +--- com.android.support:transition:28.0.0
     |    |    |    +--- com.android.support:support-annotations:28.0.0
     |    |    |    \--- com.android.support:support-compat:28.0.0 (*)
     |    |    +--- com.android.support:appcompat-v7:28.0.0
     |    |    |    +--- com.android.support:support-annotations:28.0.0
     |    |    |    +--- com.android.support:support-compat:28.0.0 (*)
     |    |    |    +--- com.android.support:collections:28.0.0 (*)
     |    |    |    +--- com.android.support:cursoradapter:28.0.0 (*)
     |    |    |    +--- com.android.support:support-core-utils:28.0.0 (*)
     |    |    |    +--- com.android.support:support-fragment:28.0.0 (*)
     |    |    |    +--- com.android.support:support-vector-drawable:28.0.0
     |    |    |    |    +--- com.android.support:support-annotations:28.0.0
     |    |    |    |    \--- com.android.support:support-compat:28.0.0 (*)
     |    |    |    \--- com.android.support:animated-vector-drawable:28.0.0
     |    |    |         +--- com.android.support:support-vector-drawable:28.0.0 (*)
     |    |    |         \--- com.android.support:support-core-ui:28.0.0 (*)
     |    |    +--- com.android.support:cardview-v7:28.0.0
     |    |    |    \--- com.android.support:support-annotations:28.0.0
     |    |    \--- com.android.support:recyclerview-v7:28.0.0
     |    |         +--- com.android.support:support-annotations:28.0.0
     |    |         +--- com.android.support:support-compat:28.0.0 (*)
     |    |         \--- com.android.support:support-core-ui:28.0.0 (*)
     |    +--- com.android.support.constraint:constraint-layout:1.1.3
     |    |    \--- com.android.support.constraint:constraint-layout-solver:1.1.3
     |    \--- android.arch.persistence.room:runtime:1.1.1
     |         +--- android.arch.persistence.room:common:1.1.1
     |         |    \--- com.android.support:support-annotations:26.1.0 -> 28.0.0
     |         +--- android.arch.persistence:db-framework:1.1.1
     |         |    +--- com.android.support:support-annotations:26.1.0 -> 28.0.0
     |         |    \--- android.arch.persistence:db:1.1.1
     |         |         \--- com.android.support:support-annotations:26.1.0 -> 28.0.0
     |         +--- android.arch.persistence:db:1.1.1 (*)
     |         +--- android.arch.core:runtime:1.1.1 (*)
     |         \--- com.android.support:support-core-utils:26.1.0 -> 28.0.0 (*)
     \--- com.google.code.gson:gson:2.8.2 -> 2.8.5

(*) - dependencies omitted (listed previously)
```

Reply:
for chucker, id just do this when adding the debugging module.

```
    implementation("debugging-dependency") {
        exclude group: 'com.github.chuckerteam.chucker', module: 'library'
    }
```